Dockerfile

```diff
@@ -167,6 +167,9 @@ RUN conda install -y -p ${CONDA_DIR} -c conda-forge r-irkernel && \
     fix-permissions $CONDA_DIR && \
     fix-permissions /home/$NB_USER
 
+# Install necessary R packages along with their dependencies
+RUN Rscript packages.R /home/$NB_USER/tmp/packages.R
```

Comment:
@j-stat Shouldn't it just be `RUN Rscript ~packages.R~ /home/$NB_USER/tmp/packages.R`